tensorflow/lite/micro/examples/hello_world/evaluate_test.cc

```diff
@@ -36,14 +50,17 @@ TfLiteStatus LoadFloatModelAndPerformInference() {
         model->version(), TFLITE_SCHEMA_VERSION);
   }
 
-  // This pulls in all the operation implementations we need
-  tflite::AllOpsResolver resolver;
+  HelloWorldOpResolver* op_resolver = nullptr;
+  if (tflite::GetOpResolver(&op_resolver) != kTfLiteOk) {
+    MicroPrintf("Could not resolve all the ops.");
+    return kTfLiteError;
+  }
```

Comment:
This should become:
```cc
HelloWorldOpResolver op_resolver;
TF_LITE_ENSURE_STATUS(RegisterOps(op_resolver));
```

We don't need additional error messages.

Reply:
done